pkgs/top-level/all-packages.nix

```diff
@@ -8606,6 +8606,7 @@ in
     inherit (darwin.apple_sdk.frameworks) Security;
   };
   rustracerd = callPackage ../development/tools/rust/racerd {
+    inherit (rustPackages_1_38_0) rustPlatform;
```

Comment:
Is this a related change?

Reply:
racerd is one of the dependencies used for rust conpletion, which broke with the recent update